Update lights readme (#493)
hyoo-ru/mam_mol: lights/readme.md

# $mol_lights

> Simple connection of the dark theme.

Reviewer: ```suggestion
Switcher between light/dark themes
```

# $mol_lights

> Simple connection of the dark theme.

Reviewer 2: Не понятно звучит. Я бы написал, что это кнопка, переключающая между светлой и тёмной темой.

## [Online demo](https://mol.hyoo.ru/#!section=demos/readme/demo=mol_lights_demo)

## Usage example

Sample usage:

```
plugins /
	<= Theme $mol_theme_auto

> tools /

Reviewer 2: Кнопку можно куда угодно помещать, не только в tools.

> 	<= lighter $mol_lights_toggle

Reviewer 2: Локальные имена компонент обычно с большой буквы начинаются.

```

Use:

**`$mol_lights_toggle`** - button for switching lights.

> **`$mol_theme_auto`** - plugin for automatic theme setting.

Reviewer 2: Это лучше вынести в доку по $mol_theme_auto, а тут лишь ссылку на неё дать.

> #### Mini wiki

Reviewer: ```suggestion
### Notes
```

Мини вики звучит не типично в проэкте. Также уровень вложенности оглавления пропущен

> Plugins are the equivalent of angular directives - components that are prefixed to the owner's home node, rather than creating their own.

Reviewer 2: prefixed -> rendered?
Вообще, это имеет смысл вынести в доку $mol_plugin

> Plugins are the equivalent of angular directives - components that are prefixed to the owner's home node, rather than creating their own.

Reviewer: ```suggestion
Plugins are components without their own DOM element, but instead use the host DOM element (like Angular's directives)
```

Немного перефразировал, лучше аналогию делать вконце, не все имеют опыт работы с ангуляром и лучше дать определение сначало как есть оно.
